Specs/Core/CartographicSpec.js

```diff
@@ -27,6 +27,16 @@ defineSuite([
         expect(c.height).toEqual(3);
     });
 
+    it('toCartesian3', function(){
```

Comment:
explain what your test is supposed to do: `toCartesian3 converts a Cartographic to a Cartesian3`